GetJobber/atlantis
# feat: add hook package with resize oberserver hook

## Motivations

This create a new package in Atlantis `hooks` - to contain reusable and importable hooks for the Atlantis library.
It also adds a reusable hook `useResizeObserver` - based on [this spike code](https://github.com/GetJobber/atlantis/pull/228/files) 


## Changes



### Added

- New `hooks` package.
- New `useResizeObserver` hook exported from the new `hooks` package.

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

* Navigate to [/components/content](https://deploy-preview-242--jobber-atlantis.netlify.app/components/content) when running this branch.
* observe that the Content component background changes depending on how wide the component is.
* Navigate to new documentation pages:
  * https://deploy-preview-242--jobber-atlantis.netlify.app/hooks/use-resize-observer
  * https://deploy-preview-242--jobber-atlantis.netlify.app/hooks
---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
